catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/criteria/contextual/ContextualEvaluator.java

```diff
@@ -197,47 +192,48 @@ public static Directory buildIndex(String fullDocument) throws IOException {
    */
   public static Directory buildIndex(String fullDocument, String[] xpathSelectors)
       throws IOException {
-    String methodName = "buildIndex";
-
-    // LOGGER.debug( XPathHelper.xmlToString( fullDocument ) );
 
     // 0. Specify the analyzer for tokenizing text.
     // The same analyzer should be used for indexing and searching
-    ContextualAnalyzer contextualAnalyzer = new ContextualAnalyzer(Version.LUCENE_30);
-
-    // 1. create the index
-    Directory index = new RAMDirectory();
-
-    // Retrieve the text from the document that can be indexed using the specified XPath
-    // selectors
-    String indexableText = getIndexableText(fullDocument, xpathSelectors);
-
-    // Create an IndexWriter using the case-insensitive StandardAnalyzer
-    // NOTE: the boolean arg in the IndexWriter constructor means to create a new index,
-    // overwriting any existing index
-    try (IndexWriter indexWriter =
-        new IndexWriter(index, contextualAnalyzer, true, IndexWriter.MaxFieldLength.UNLIMITED)) {
-      logTokens(indexWriter.getAnalyzer(), FIELD_NAME, fullDocument, "ContextualAnalyzer");
-
-      // Add the indexable text to the case-insensitive index writer, assigning it the
-      // "case-insensitive" field name
-      addDoc(indexWriter, FIELD_NAME, indexableText);
-    }
-    CaseSensitiveContextualAnalyzer caseSensitiveStandardAnalyzer =
-        new CaseSensitiveContextualAnalyzer(Version.LUCENE_30);
-
-    // Create a second IndexWriter using the custom case-sensitive StandardAnalyzer
-    // NOTE: set boolean to false to append the case-sensitive indexed text to the existing
-    // index (populated by first IndexWriter)
-    try (IndexWriter csIndexWriter =
-        new IndexWriter(
-            index, caseSensitiveStandardAnalyzer, false, IndexWriter.MaxFieldLength.UNLIMITED)) {
-
-      // Add the indexable text to the case-sensitive index writer, assigning it the
-      // "case-sensitive" field name
-      addDoc(csIndexWriter, CASE_SENSITIVE_FIELD_NAME, indexableText);
+    try (ContextualAnalyzer contextualAnalyzer = new ContextualAnalyzer(Version.LUCENE_30)) {
+
+      // 1. create the index
+      try (Directory index = new RAMDirectory();
```

Comment:
✏️  Can we add a \\\\ after the ( just to push the rest on the next line and to make it more obvious that it is part of a block?